runtime/src/bank.rs

```diff
@@ -9301,29 +9270,22 @@ pub(crate) mod tests {
         // verify that there's inflation
         assert_ne!(bank1.capitalization(), bank0.capitalization());
 
-        // verify the inflation is represented in validator_points *
-        let paid_rewards = bank1.capitalization()
-            - bank0.capitalization()
-            - bank1_sysvar_delta()
-            - new_epoch_sysvar_delta();
-
-        let rewards = bank1
-            .get_account(&sysvar::rewards::id())
-            .map(|account| from_account::<Rewards, _>(&account).unwrap())
-            .unwrap();
+        // verify the inflation is represented in validator_points
+        let paid_rewards = bank1.capitalization() - bank0.capitalization() - bank1_sysvar_delta();
+        let (validator_rewards, _, _, _) =
+            bank1.inflation_info_during_epoch(bank0.capitalization(), bank0.epoch());
```

Comment:
This looks good but can you add a comment saying that this assumes that no new builtins / precompiles were activated in bank1? If they were activated, bank1's capitalization would be increased before calculating inflation rewards